### ISSUE TITLE
Removing ML workspace fails

### ISSUE THREAD

**Issue by strokovnjaka** [microsoft/vscode-tools-for-ai#1402]
Does this occur consistently? yes
Repro steps:
1. create workspace as per [tutorial](https://docs.microsoft.com/en-us/azure/machine-learning/tutorial-train-deploy-image-classification-model-vscode)
2. try to create GPU cluster per same tutorial (which, probably not important for this bug, fails due to `The specified subscription has a Standard NC family vCPU quota of 0 and cannot accomodate for at least 1 requested managed compute nodes which maps to 6 vCPUs.`)
3. try to delete in step 1 created workspace, select `With associated resources`
4. observe error

Action: vscodeai.azureml.workspace.remove
Error type: TypeError
Error Message: Cannot read property 'match' of null

Version: 0.6.27
OS: darwin
OS Release: 19.6.0
Product: Visual Studio Code
Product Version: 1.62.3
Language: en

<details>
<summary>Call Stack</summary>

```
v extension.js:2:4627128
Object.<anonymous> extension.js:2:4626971
extension.js:2:4621661
n extension.js:2:4621406
Object.y [as getSubscriptionFromUri] extension.js:2:4626929
m.<anonymous> extension.js:2:4308975
extension.js:2:4305198
s extension.js:2:4304943
m.deleteContainerRegistry extension.js:2:4308872
m.<anonymous> extension.js:2:4306751
a extension.js:2:4305000
processTicksAndRejections task_queues.js:93:5
```

</details>

**Comment by sevillal** [comment]
Thanks @strokovnjaka  for bringing this to our attention. 

I was able to repro this issue, there is a bug in the extension when trying to  delete the workspace, specifically the container registry the following property 'workspace.properties.containerRegistry' is unavailable (null). We will plan this fix and update you on when would be ready.

**Comment by sevillal** [comment]
Should be fixed in our next release 0.6.28
